guava/src/com/google/common/eventbus/SubscriberRegistry.java

```diff
@@ -174,6 +199,20 @@ void unregister(Object listener) {
     return methodsInListener;
   }
 
+  /**
+   * Returns all subscribers for the given listener grouped by the type of event they subscribe to.
+   */
+  private Multimap<Class<?>, Subscriber> findAllSubscribers(Object listener, Executor executor) {
```

Comment:
This can be just a `Map` since you do not need `Multimap` functionalities.

Reply:
Keep the same semantics as the original method, eg:
```java
private Multimap<Class<?>, Subscriber> findAllSubscribers(Object listener)
```

Reply:
@liach It's okay?

Reply:
Yeah, you can ignore my old comment.